vulnerability-analyzer/src/main/java/org/acme/analyzer/InternalAnalyzer.java

```diff
@@ -62,12 +64,12 @@ public AnalyzerIdentity getAnalyzerIdentity() {
     private final VulnerableSoftwareRepository vulnerableSoftwareRepository;
 
     @Inject
-    @CacheName("internal")
+    @CacheName("internalCache")
```

Comment:
For this name change, please also change the cache name in application.properties